### PR TITLE
chore: support tron address validation

### PR DESCRIPTION
**Pull request by jerrylaloan**
## Description of the changes

- updated address validation 
- added TRON chain configuration

### CODE REVIEW

File: packages/currency/src/chains/tron/data/tron.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const chainId = 'mainnet';
```
Comment (alexandre-abrioux) [inline packages/currency/src/chains/tron/data/tron.ts:1]: Found here: https://medium.com/tronnetwork/how-to-adapt-to-optimization-of-the-return-value-of-chainid-opcode-in-tvm-6160029b64a0
```suggestion
export const chainId = 728126428;
```

Comment (jerrylaloan) [reply packages/currency/src/chains/tron/data/tron.ts:1]: ah, good find! 👍

File: packages/currency/src/currencyManager.ts [hunk 2]
```diff
@@ -246,6 +246,8 @@ export class CurrencyManager<TMeta = unknown> implements ICurrencyManager<TMeta>
       case RequestLogicTypes.CURRENCY.ERC777:
         if (NearChains.isChainSupported(currency.network)) {
           return isValidNearAddress(address, currency.network);
+        } else if (DeclarativeChains.isChainSupported(currency.network)) {
```
Comment (benjlevesque) [inline packages/currency/src/currencyManager.ts:249]:
```suggestion
        } else if (currency.network === "tron") {
```

**Review by benjlevesque — APPROVED** [review]
I'm uneasy with

**Review by alexandre-abrioux — APPROVED** [review]

**Review by yomarion — APPROVED** [review]

**Review by MantisClone — COMMENTED** [review]
Looks good to me :+1:

**Pull request merged**